app/factories/bulkrax/object_factory.rb

```diff
@@ -70,6 +71,12 @@ def run!
     end
 
     def update
+      if @importer.import_relationships_only
+        logger = Rails.logger
+        logger.info("Skipping #{klass} #{object.id} (#{Array(attributes[work_identifier]).first})")
```

Comment:
This is great to have logging; wondering if there's more information to include.  `"Skipping updating metadata for #{klass} ID=#{object.id} (#{Array(attributes[work_identifier]).first})"`